libraries/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -235,7 +235,9 @@
     #define EMPTY_ELSE_MARKER
 #endif
 
-#define IOT_MQTT_SERVER_MAX_CLIENTID_LENGTH                    ( ( uint16_t ) 23 ) /**< @brief Optional maximum length of client identifier specified by MQTT 3.1.1. */
+#define IOT_MQTT_SERVER_MAX_CLIENTID_LENGTH                    ( ( uint16_t ) 23 )          /**< @brief Optional maximum length of client identifier specified by MQTT 3.1.1. */
+#define IOT_MQTT_SERVER_MAX_PUBLISH_PAYLOAD_LENGTH             ( ( uint32_t ) ( 1 << 28 ) ) /**< @brief Maximum publish payload length supported by MQTT 3.1.1. */
```

Comment:
Could you change this to `268435456`.  It's easier for the reader than `1<<28`.

Same for `1<<17` below.

Reply:
Changed